src/TimeGrid.jsx

```diff
@@ -77,9 +80,23 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
+    }
+
     //this.checkOverflow()
   }
 
+  componentWillReceiveProps(nextProps) {
+    // When paginating, reset scroll
+    if (+nextProps.start !== +this.props.start) {
```

Comment:
Good point about the hidden calendar. What do you suggest? Just skip this `if` or check if `scrollTime` has changed too?